themes/clr-installer.theme

```diff
@@ -71,45 +93,52 @@ ErrorLabel.Text = red bold
 
 Menu.ButtonBack=blue
 Menu.ButtonText=white
+
 Menu.ButtonActiveBack=cyan
 Menu.ButtonActiveText=black
+
 Menu.ButtonShadowBack=black
+
 Menu.ButtonDisabledText=white bold
 Menu.ButtonDisabledBack=red
 
-
+// disk partitioning
 Part.ButtonBack=blue
 Part.ButtonText=white
+
 Part.ButtonActiveBack=cyan
-Part.ButtonActiveText=black
+Part.ButtonActiveText=blue
+
 Part.ButtonShadowBack=black
-Part.ButtonDisabledText=white bold
-Part.ButtonDisabledBack=red
+
+Part.ButtonDisabledText=blue
+Part.ButtonDisabledBack=black
 
 
 //----- Main menu tab theme ------------------
-Tab.ButtonBack=white
-Tab.ButtonText=blue bold
+Tab.ButtonBack=black
+Tab.ButtonText=white bold
 
-Tab.ButtonActiveBack=blue
-Tab.ButtonActiveText=white bold
+Tab.ButtonActiveBack=cyan
+Tab.ButtonActiveText=blue bold
 
 Tab.ButtonShadowBack=black
+
 Tab.ButtonDisabledText=white bold
 Tab.ButtonDisabledBack=red
 
-Tab.ViewBack = white
-Tab.ViewText = blue
+Tab.ViewBack = black
+Tab.ViewText = white
 
 // ---- Main menu items ---------------------
 Main.MenuText = white bold
 Main.MenuBack = blue
 
 Main.MenuActiveText=blue bold
-Main.MenuActiveBack=white
+Main.MenuActiveBack=cyan
 
-Main.MenuContentText=white
+Main.MenuContentText=cyan bold
 Main.MenuContentBack=blue
 
 Main.MenuContentActiveText=blue
-Main.MenuContentActiveBack=white
+Main.MenuContentActiveBack=cyan
```

Comment:
The Checkbox for a Network Interface is still Magenta when selected instead of Cyan.

Reply:
Same color issues for the Checkboxes used for Bundle selection; Magenta instead of Cyan.

Reply:
Also the Bullet List choose for Kernels is still Magenta

Reply:
yeah! missed that one. Thanks.

Reply:
At line 75 I think you need to add:
```
+ControlText = white
+ControlDisabledBack = blue
+ControlDisabledText = white
+ControlActiveBack = cyan
+ControlActiveText = blue bold
```

Reply:
done. thanks.